tests/test_autoencoderkl_maisi.py

```diff
@@ -0,0 +1,227 @@
+# Copyright (c) MONAI Consortium
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#     http://www.apache.org/licenses/LICENSE-2.0
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from __future__ import annotations
+
+import unittest
+
+import torch
+from parameterized import parameterized
+
+from monai.networks import eval_mode
+from monai.utils import optional_import
+from tests.utils import SkipIfBeforePyTorchVersion
+
+tqdm, has_tqdm = optional_import("tqdm", name="tqdm")
+_, has_einops = optional_import("einops")
+_, has_generative = optional_import("generative")
+
+if has_generative:
+    from monai.apps.generation.maisi.networks.autoencoderkl_maisi import AutoencoderKlMaisi
+
+device = torch.device("cuda:0" if torch.cuda.is_available() else "cpu")
+
+
+CASES_NO_ATTENTION = [
```

Comment:
Could you please add more test cases to include 2d cases and more parameters here?
You can add them in a for loop like this: https://github.com/Project-MONAI/MONAI/blob/e801540166655fb602df8f749089db2a4cabc0de/tests/test_selfattention.py#L28

Thanks!

Reply:
MAISI model is dedicated for 3D images, which is not designed for 2D images since 2D image normally do not have out-of-memory issue.

Reply:
My bad, just ignore it. Thanks.